enterprise/backend/src/metabase_enterprise/sso/api/interface.clj

```diff
@@ -35,8 +35,10 @@
   sso-backend)
 
 (defn- throw-not-configured-error []
-  (throw (ex-info (str (tru "SSO has not been enabled and/or configured"))
-                  {:status-code 400})))
+  (throw
+    (ex-info "SSO has not been enabled and/or configured"
```

Comment:
I removed it because I didn't know what it did and I just wanted to worry about the strings 😅  from the usage I had assumed it was a string.format thing but now I know

Reply:
so tl;dr i'll re-add the translations